### PR TITLE
Anchor Platform: Add Event documentation

### PR DESCRIPTION
**Pull request by philipliu**
This adds the integration guide for Anchor Platform's event service.

### CODE REVIEW

File: docs/anchoring-assets/anchor-platform/events/integration.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Anchor Platform signs the callback requests it sends to client applications. The signature is included in the `Signature` header of the request. The callback URL signature specification can be found in the corresponding SEP protocol specifications.
+
+- [SEP-0006](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#url-callback-signature)
+- [SEP-0024](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#url-callback-signature)
+- [SEP-0031](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0031.md#url-callback-signature)
+
+### As a Business Server
+
+In addition to SEP transaction status updates, business servers can receive events about SEP-31 quote creation or SEP-12 customer information updates. The schema of the event data will depend on the type of event being sent. Visit the [Event API documentation](/api/anchor-platform/callbacks/post-event) for more information about the schema of the event data.
```
Comment (Ifropc) [inline docs/anchoring-assets/anchor-platform/events/integration.mdx:84]: When do we send event for SEP-12 customer updated?

File: docs/anchoring-assets/anchor-platform/events/integration.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+</CodeExample>
+
+This will enable the event processor to start processing events from `TRANSACTION` topic. In this example, the event processor will send events to client and business server callback endpoints.
+
+Next, the event service's Kafka producer need to be configured using the `event.queue` section of the configuration file. The following is an example of how to configure the event service's Kafka producer:
+
+<CodeExample>
+
+```yaml
```
Comment (Ifropc) [inline docs/anchoring-assets/anchor-platform/events/integration.mdx:36]: In all other places we use env variables for configs. 
I know it's easier with yaml but I think for at least v2 of the document we should use env everywhere for consistency

File: docs/anchoring-assets/anchor-platform/events/integration.mdx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  secret: bf7c2c5b-5c76-4974-a318-ac9c4a2b8514
+  auth:
+  type: api_key
+  jwt:
+    http_header: X-Api-Key
+```
+
+:::caution
+
+The `http_header` is configurable as of version `2.5.2` and earlier. The default value is `Authorization` for JWT and `X-Api-Key` for API key.
```
Comment (Ifropc) [inline docs/anchoring-assets/anchor-platform/events/integration.mdx:123]: Should it say that it's configurable, but not working as intended? (Judging by the bug description)

File: docs/anchoring-assets/anchor-platform/events/delivery.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+:::
+
+Next subsections will describe the delivery guarantees from the client and the business server perspective.
+
+### Client Delivery Guarantees
+
+For each client, the event service will attempt to deliver each event up to three times with an exponential backoff. If the event is not delivered after three attempts due to HTTP 4xx or 5xx errors, the event will be skipped. If the client is not reachable after three attempts, the event service will no longer attempt to deliver any events to that client.
+
+### Business Server Delivery Guarantees
+
```
Comment (Ifropc) [inline docs/anchoring-assets/anchor-platform/events/delivery.mdx:23]: We plan to change that, right? I think it should be noted here

File: docs/anchoring-assets/anchor-platform/events/integration.mdx (hunk 3 above)
Comment (philipliu) [reply docs/anchoring-assets/anchor-platform/events/integration.mdx:123]: Oops, I meant _not_ configurable. Thanks for catching this.

File: docs/anchoring-assets/anchor-platform/events/integration.mdx (hunk 1 above)
Comment (philipliu) [reply docs/anchoring-assets/anchor-platform/events/integration.mdx:84]: On every SEP-12 customer PUT. I'll update it.

Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/events/integration.mdx:84]: Don't we already receive a callback on SEP-12 PUT to the business server? Why do we want to also receive an event associated with it?

Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/events/integration.mdx:115]: I was just curious why we added it in a first place. Got it, thanks!

**Review by Ifropc — APPROVED** [review]

**Pull request merged**